ui/packages/catalog-ui-search/src/main/webapp/react-component/sharing/container.tsx

```diff
@@ -167,7 +171,7 @@ export class Sharing extends React.Component<Props, State> {
       e => e.value !== '' && e.category === Category.User
     )
 
-    if (this.state.isWorkspace && guest[0].access === 0) {
+    if (this.state.type === 'workspace' && guest[0].access === 0) {
```

Comment:
consider using an enum/constants for metacard types (both on the java side and js side)

Reply:
done